features/title.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <title>
-description: The `<title>` element TODO.
+description: The `<title>` element sets the title of the document and exposes it to users via the UI of the browser or app they're using and through assistive technology.
 spec: https://html.spec.whatwg.org/multipage/semantics.html#the-title-element
 group: html
 compat_features:
```

Comment:
Somehow GitHub doesn't show me the content of the dist file here, only that it moved. So I didn't see this the first time I reviewed. But it seems like adding the two `api.*` keys in here causes the feature to appear as supported starting with Safari 3 only.
It's OK since the feature is baseline high either way, but adding `compute_from: html.elements.title` would make the feature start at the earliest version of all browsers, which would make more sense here.

Reply:
done!